AsyncDisplayKit/Details/ASChangeSetDataController.mm

```diff
@@ -66,6 +66,8 @@ - (void)endUpdatesAnimated:(BOOL)animated completion:(void (^)(BOOL))completion
     [self invalidateDataSourceItemCounts];
     [_changeSet markCompletedWithNewItemCounts:[self itemCountsFromDataSource]];
     
+    ASDataControllerLogEvent(self, @"submitting: %@", _changeSet);
```

Comment:
Let's go with `triggeredUpdate:` and `finishedUpdate:` to be in line with the other `didThing` log events.